pkg/utils/integration/controller.go

```diff
@@ -259,7 +265,7 @@ func (h *SuiteController) CreateIntegrationPipelineRun(applicationSnapshotName,
 	return testpipelineRun, err
 }
 
-func (h *SuiteController) CreateIntegrationTestScenario(applicationName, namespace, bundleURL, pipelineName string) (*integrationv1alpha1.IntegrationTestScenario, error) {
+func (h *SuiteController) CreateIntegrationTestScenario(applicationName, namespace string) (*integrationv1alpha1.IntegrationTestScenario, error) {
```

Comment:
More than one IntegrationTestScenario need to be created in the future.  The two params will be needed at that time.